lighthouse-core/audits/bootup-time.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const DevtoolsTimelineModel = require('../lib/traces/devtools-timeline-model');
+const WebInspector = require('../lib/web-inspector');
+const Util = require('../report/v2/renderer/util.js');
+
+const group = {
```

Comment:
could pull these two objects into a `lib/traces/trace-event-groups.js` or something so that they can be used across audits